### ISSUE TITLE
Blocked on Anti-virus solutions. (Windows-x64, LDC2-1.23.0)

### ISSUE THREAD

**Issue by ZHANITEST** [ldc-developers/ldc#3563]
Hi. Title is description.

* Target: [ldc2-1.23.0-windows-x64.7z](https://github.com/ldc-developers/ldc/releases/tag/v1.23.0), `bin/libcurl.dll`
* VirusTotal : [23 engines detected ](https://www.virustotal.com/gui/file/5b965af2d2df7f290addb855a4f8a8da06849104d04045b83756b231af81f029/detection)

I don't know why. Maybe misdiagnosis or not?

**Comment by kinke** [comment]
Duplicate of https://github.com/ldc-developers/ldc/issues/3472.
